pyrs/core/summary_generator_stress.py

```diff
@@ -129,9 +129,14 @@ def _write_summary_csv_column_names(handle):
         def _write_summary_csv_body(handle):
 
             def _write_number(number) -> str:
-                if math.isnan(number):
+                if abs(number-math.floor(number)) <= 1e-15:
```

Comment:
I like that you're checking for numbers that are close enough to integers

Reply:
no love for `1.00000000000000000000000000000` ?